scripts/helm/publish-chart-yaml.sh

```diff
@@ -49,7 +49,7 @@ branch_chart_version()
     echo "0.0.0"
   elif [ "$CHECK_BRANCH" == "main" ]; then
     # Main has no meaningful version, other than the date-time
-    echo "0.0.0-$DATE_TIME"
+    echo "0.0.0-main"
```

Comment:
hmm don't think we want to do this right? We want to support both ways, only if date-time is not set we should use the main postfix.